tests/files/packet_centos7-flannel-addons.yml

```diff
@@ -6,7 +6,6 @@ mode: ha
 # Kubespray settings
 kubeadm_control_plane: true
 kubeadm_certificate_key: 3998c58db6497dd17d909394e62d515368c06ec617710d02edea31c06d741085
-kube_proxy_mode: iptables
```

Comment:
wow
iptables mode kube_proxy in 1.17.4 not support flannel ???

Reply:
With `kube_proxy_mode: iptables` there is a problem to access the NodePort service for netchecker. The manual packet_ubuntu18-flannel-containerd CI job has the same error (with `kube_proxy_mode: iptables`). I suggested to @floryut to remove that line as a troubleshooting step (to see if there are other problems we can address).